src/models/video/index.ts

```diff
@@ -26,6 +28,7 @@ export class Video {
     this.id = data.id;
     this.metadata = data.metadata;
     this.hls = data.hls;
+    this.source = data.source;
```

Comment:
similar q as python sdk pr. why indexed_at is missing here? should we add that field too?

Reply:
just added `indexed_at` in video object, I missed!